scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -613,6 +621,7 @@ class FossIdTest : WordSpec({
             }
         }
 
+        @Suppress("LargeClass")
```

Comment:
This annotation needs to go to the *class*, so before line 97.

Reply:
Done